### DISCUSSION TITLE
Full Text Search Postgres is case insensitive by default

### DISCUSSION THREAD

**Discussion by roshcagra** [prisma/prisma#25881] [Q&A]
### Bug description

Hi! I'm using Prisma Client 6.0.1 with Postgres 16 and it looks like full text search is defaulting to non-case sensitive? I was wondering if this is a bug or the expectation, because we need case-sensitive search. Thanks!

### How to reproduce

Perform a search using `search` on a postgres server with `fullTextSearchPostgres` enabled on the Prisma client.

### Expected behavior

I would expect the search to be case-sensitive since that's the default for other Prisma functions on Postgres.

### Prisma information

```prisma
generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["tracing", "omitApi", "fullTextSearchPostgres"]
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}
...
```

```ts
await prisma.myModel.findFirst({
      where: {
        AND: [
          {
            myField: {
              search: "!Case"
            },
          },
          {
            myField: {
              not: null,
            },
          },
        ],
      }
    });
```

### Environment & setup

- OS: macOS
- Database: PostgreSQL 16
- Node.js version: 20.10.0

### Prisma Version

```
6.0.1
```

**Comment by ludralph** [comment]
Hi @roshcagra 

The default behavior of full-text search in PostgreSQL is case insensitive. The behavior you're experiencing is the expected behavior for PostgreSQL full-text search, rather than a bug in Prisma. If you need case-sensitive search, you might need to use [raw SQL](https://www.prisma.io/docs/orm/prisma-client/using-raw-sql) queries to accomplish this.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
